packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts

```diff
@@ -231,6 +230,8 @@ const init = async () => {
     return 0;
   }
 
+  const fd = fs.openSync(filepath, 'a');
```

Comment:
`node:fs/promises` supports async/await for file io, not that it really matters here.